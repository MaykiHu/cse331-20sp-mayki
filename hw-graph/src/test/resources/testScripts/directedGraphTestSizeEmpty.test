#Create a graph
CreateGraph graph1

#Gets size of an empty graph
GetSize graph1