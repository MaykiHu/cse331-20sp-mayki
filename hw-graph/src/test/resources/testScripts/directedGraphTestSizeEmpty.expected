#Create a graph
created graph graph1

#Gets size of an empty graph
size of graph1 is 0